spaces/S000118/properties/P000204.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000118
+property: P000204
+value: true
+---
+
+The origin is a cut point since the complement of the origin can be written as a disjoint union of nonempty open sets, e.g., $([1, \infty) \cap \omega) \times (\{0\} \cup \{\frac{1}{n}\}_{n=2}^\infty)$ and $([1, \infty) \cap \omega) \times \{0\}$; and, moreover, the space itself is known to be connected (see {S118|P36}).
```

Comment:
```suggestion
The origin is a cut point since the complement of the origin can be written as a disjoint union of nonempty open sets, e.g., $([1, \infty) \cap \omega) \times (\{0\} \cup \{\frac{1}{n}\}_{n=2}^\infty)$ and $([1, \infty) \cap \omega) \times \{0\}$; and moreover, {S118|P36}.
```
simplification